docs/users/globebrowsing/geojson-layers.md

```diff
@@ -0,0 +1,160 @@
+
+The following table summarizes the available keys and properties. Note that the property identifier is also always a valid key in the GeoJson file.
+
+| Property Identifier | Accepted keys (in .geojson file) | Value                                    | Description |
+| ------------------- |--------------------------------- | ---------------------------------------- | ----------- |
+| Name                | `name`                           | string                                   | This will be the name for the feature that shows in the "Features" menu in the UI |
+| Description         | `description`                    | string                                   | (Not used anywhere right now, but will be shown in the UI in the future) |
+| Opacity             | `opacity`                        | float, [0-1]                             | The opacity that will be used for drawing lines and points |
+| Color               | `color`, `stroke`                | RGB (vec3, [0-1]) or hex value (string)  | The color that will be used for drawing lines and points. For points, the color will be blended with the color of the point texture |
+| FillColor           | `fill`, `fill-color`             | RGB (vec3, [0-1]) or hex value (string)  | The color that will be used for drawing filled polygons and extruded geometry |
+| FillOpacity         | `fill-opacity`                   | float, [0-1]                             | The opacity that will be used for drawing filled polygons and extruded geometry |
+| Extrude             | `extrude`                        | boolean (`true`/`false`)                 | If true, extrude the geometry to intersect the globe. Lines/polygons will be extruded with polygons, and points with lines |
+| AltitudeMode        | `altitudeMode`                   | string (`"absolute"` or `"relativeToGround"`) | Decides how any height values of the geo coordinates should be interpreted. "Absolute" means that the height is interpreted as the height above the reference ellipsoid (sea level), while "RelativeToGround" takes the height map into account |
+| **Points Specific**          |                   |                              |  |
+| PointSize           | `point-size`                     | float (> 0)                              | A value that will be used to decide the size of rendered points. The size will be scaled based on the bounding sphere of the globe   |
+| PointTexture        | `texture`, `sprite`, `point-texture` | string (path to a texture)           | A texture to be used for rendering points |
+| PointTextureAnchor  | `point-anchor`, `anchor`         | string (`"bottom"` or `"center"`)        | Decides the placement of the point texture in relation to the position. Default is a the bottom of the texture, but it can also be put at the center |
```

Comment:
```suggestion
| PointTextureAnchor  | `point-anchor`, `anchor`         | string (`"bottom"` or `"center"`)        | Decides the placement of the point texture in relation to the position. Default is at the bottom of the texture, but it can also be put at the center |
```